test/controllers/update/update.filing.date.controller.spec.ts

```diff
@@ -93,6 +94,8 @@ describe("Update Filing Date controller", () => {
       expect(resp.text).toContain(BACK_LINK_FOR_UPDATE_FILING_DATE);
       expect(resp.text).toContain(saveAndContinueButtonText);
       expect(resp.text).not.toContain(PAGE_TITLE_ERROR);
+      expect(resp.text).not.toContain(UPDATE_DATE_OF_UPDATE_STATEMENT_TEXT);
```

Comment:
Would have thought that we should be checking that the response text **does** contain the new test? Same with the other tests.